docs/guides/livestreaming/webhook.mdx

```diff
@@ -60,52 +82,72 @@ this account.
 
 `DEL /api/webhook/:webhookID`: Delete a webhook
 
-### Types of Events
-
-<Table
-  data={[
-    {
-      name: "stream.started",
-      description:
-        "The parent stream object's isActive value is marked as true and the .m3u8 HLS URL works",
-    },
-    {
-      name: "stream.idle",
-      description:
-        "The parent stream object's isActive value should be marked as false and the .m3u8 HLS URL no longer works",
-    },
-    {
-      name: "recording.ready",
-      description: "This fires when a recording is ready to be downloaded",
-    },
-    {
-      name: "recording.started",
-      description: "This fires when recording has started on an active stream",
-    },
-    {
-      name: "recording.waiting",
-      description:
-        "This fires after a stream with recording on has concluded and is not yet ready to be downloaded. Typically it takes 5 minutes for recordings to be ready for download.",
-    },
-    {
-      name: "multistream.connected",
-      description:
-        "This fires when we've successfully connected to the multistream target",
-    },
-    {
-      name: "multistream.error",
-      description:
-        "This fires when we've encountered an error either while attempting to connect to the third party streaming service or while broadcasting.",
-    },
-    {
-      name: "multistream.disconnected",
-      description:
-        "This fires when we are no longer sending video to the multistream target.",
-    },
-  ]}
-/>
-
-### Retries
+## Types of Events
+
+| Name                     | Description                                                  |
+| ------------------------ | ------------------------------------------------------------ |
+| stream.started           | The parent stream object's isActive value is marked as true and the .m3u8 HLS URL works |
+| stream.idle              | The parent stream object's isActive value should be marked as false and the .m3u8 HLS URL no longer works |
+| recording.ready          | This fires when a recording is ready to be downloaded        |
+| recording.started        | This fires when recording has started on an active stream    |
+| recording.waiting        | This fires after a stream with recording on has concluded and is not yet ready to be downloaded. Typically it takes 5 minutes for recordings to be ready for download. |
+| multistream.connected    | This fires when we've successfully connected to the multistream target |
+| multistream.error        | This fires when we've encountered an error either while attempting to connect to the third party streaming service or while broadcasting. |
+| multistream.disconnected | This fires when we are no longer sending video to the multistream target. |
+
+## Retries
 
 Currently, if a webhook fails, Livepeer will attempt to deliver the webhook for
 ~15 minutes with an exponential back off.
```

Comment:
Move section to line 31, after "About Webhooks" section. Might be a good idea to introduce the different types of events to use before introducing on how to configure them.